jumpscale/packages/threebot_deployer/chats/threebot.py

```diff
@@ -357,7 +357,7 @@ def _deploy(self):
                 ## Container logs
                 log_config = j.core.config.get("LOGGING_SINK", {})
                 if log_config:
-                    log_config["channel_name"] = self.solution_name
+                    log_config["channel_name"] = f"{self.threebot_name}-{self.SOLUTION_TYPE}-{self.solution_name}"
```

Comment:
I think we can do `.replace("_", "-")`  for for the owner too ?

also the `self.SOLUTION_TYPE.lower()` it always starts with capital letter